upgrade_policy/quorum_upgrade/sources/quorum_upgrade_policy.move

```diff
@@ -289,15 +291,15 @@ module quorum_upgrade_policy::quorum_upgrade_policy {
     /// Vote in favor of an upgrade, aborts if the voter is not for the proposed
     /// upgrade or if they voted already, or if the upgrade was already performed.
     public fun vote(
-        proposal: &mut ProposedUpgrade, 
+        proposal: &mut ProposedUpgrade,
         voter: &mut VotingCap,
         ctx: &TxContext,
     ) {
         assert!(proposal.proposer != @0x0, EAlreadyIssued);
         assert!(proposal.upgrade_cap == voter.upgrade_cap, EInvalidVoterForUpgrade);
         let voter_id = object::id(voter);
         assert!(
-            !vec_set::contains(&proposal.current_voters, &voter_id), 
+            !vec_set::contains(&proposal.current_voters, &voter_id),
```

Comment:
Since you refactor, we could also move 2024 😄 
```suggestion
            !proposal.current_voters.contains(&voter_id),
```

Reply:
Yes :)